alibi_detect/saving/schemas.py

```diff
@@ -367,6 +367,9 @@ class KernelConfig(CustomBaseModelWithKwargs):
     should match :py:func:`~alibi_detect.utils.tensorflow.kernels.sigma_median`. If `None`, it is set to
     :func:`~alibi_detect.utils.tensorflow.kernels.sigma_median`.
     """
+    # Validators
+    _validate_flavour = validator('flavour', allow_reuse=True, pre=False)(validate_framework)
+    _coerce_sigma2tensor = validator('sigma', allow_reuse=True, pre=False)(coerce_2tensor)
```

Comment:
Almost all of our public API expects `list`'s or `np.ndarray`'s. However, a peculiarity of our `GaussianRBF` kernels is that they are public, but expect `sigma` to be a `tf.Tensor` or `torch.Tensor`. Previously we were coercing `sigma` into a `np.ndarray` (via the `NDArray` pydantic type). It wasn't strictly correct to pass this to the tensorflow `GaussianRBF`, however we got away with it since `sigma` was coerced to a `tf.Tensor` internally.  The pytorch `GaussianRBF` does not do this, hence passing `sigma` as an `np.ndarray` fails. Therefore, the `coerce_2tensor` validator has been added to coerce `sigma` to the correct `Tensor` depending on the value of `flavour`. Depending on @Srceh's kernel updates we can retain this functionality or remove as needed.